src/test/ReverseTest.java

```diff
@@ -0,0 +1,27 @@
+import com.company.reverse.Reverse;
```

Comment:
test classes should be in same package as class under the test

Reply:
Done